### PR TITLE
sortable tasks, and update order on change

### PR DESCRIPTION
**Pull request by colinyoung87**

### CODE REVIEW

File: app/assets/javascripts/directives/task_item.js [hunk 1]
```diff
@@ -41,6 +41,17 @@ angular.module("OpenHq").directive("taskItem", function($rootScope, Task, Confir
           $scope.task.update();
         }
       });
+
+      $(".tasks ul.sortable").sortable({
```
Comment (phawk) [inline app/assets/javascripts/directives/task_item.js]: Is this gonna run a whack of times?

File: app/assets/javascripts/directives/task_item.js [hunk 2]
```diff
@@ -41,6 +41,17 @@ angular.module("OpenHq").directive("taskItem", function($rootScope, Task, Confir
...
+        items: "li",
+        update: function() {
+          var $this = $(this),
+              story_id = $this.attr('data-story-id'),
```
Comment (phawk) [inline app/assets/javascripts/directives/task_item.js]: you should be able to get the story slug from route params or the ID from $scope

Comment (colinyoung87) [inline app/assets/javascripts/directives/task_item.js]: oh yeah! whoops

File: app/assets/javascripts/directives/task_item.js (hunk 1 above)
Comment (colinyoung87) [inline app/assets/javascripts/directives/task_item.js]: yeah... thats the way it was before though - i think it automatically clears any old sortable stuff when you run it again. Will look into it.

**Pull request merged**